### PR TITLE
More Cyberware

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Fixes being able to install organ modules in the wrong slot and having them just work:tm:  
Adds in 6!! new cyber implants
S.M.H Enhancer: Helps not sanity not die as fast and store more
Cortex Support Cyberware: Massively (500%) more passive sanity gain - This is not as huge of a game changer as this is for passive sanity, meaning rather then 0.2 healed its 1 healed, out of 200. Meaning its not that powerful as you would think
Imbedded pouch: Small pouch holds 1 less and has higher then normal costs (eats stats)
Sanguin-Stash: Lets the installed warer hold 100u more blood
S.N.A.C.K System: Hold more hunger, and loss less passively
NSA Banker: Gives 30 NSA (In terms of viv)

Addds 3 new nodes for said implants to RnD
6750 more points for a total

image
image

Dif is having a moment

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
According to Chef and Raptor the values for stuff might be off

**Comment by cdb-is-not-good** [comment]
I'm worried about the sanity one being useable to seriously power game inspiration

**Comment by benj8560** [comment]
> I'm worried about the sanity one being useable to seriously power game inspiration

had a poke and tested it (as one of those sanity power gamers) it's not as impressive as 500% sounds. The passive sanity gain rate is pretty damn low and useless as is. Is it good for grinding stats? yes. Is it even mildly better then the other current metas? no not remotely.

File: code/game/objects/items/devices/organ_module/passive/food.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/obj/item/organ_module/snack_system
+	name = "S.N.A.C.K System"
+	desc = "The Storage Nutrience: Advanced Cal Keeper, a state of the art addition to the large intestant track, \
+	both as a helper prevent nutrition loss and store a bunch of healthy nutrition. The designs are credited to two SI students."
+	matter = list(MATERIAL_WOOD = 1, MATERIAL_PLASTIC = 12, MATERIAL_GLASS = 5, MATERIAL_SILVER = 1)
```
[the remark this answers is not in the document]
Comment (Trilbyspaceclone) [reply code/game/objects/items/devices/organ_module/passive/food.dm:5]: Bento box

**Comment by Trilbyspaceclone** [comment]
Lets see how this goes for now

**Pull request merged**